ResourceStatusSystem/scripts/dirac-rss-set-status.py

```diff
@@ -46,13 +43,8 @@ def registerUsageMessage():
   '''
     Takes the script __doc__ and adds the DIRAC version to it
   '''
-
-  hLine = '  ' + '=' * 78 + '\n'
```

Comment:
@fstagni: When dumping the docstring to rst the equal signs cause warnings, because they  resemble the header syntax, so I removed them from the docstring.